Buildings/Fluid/DXSystems/Heating/BaseClasses/CoilDefrostTimeCalculations.mo

```diff
@@ -0,0 +1,145 @@
+within Buildings.Fluid.DXSystems.Heating.BaseClasses;
+block CoilDefrostTimeCalculations
+  "Calculates defrost curve value at given temperature and mass flow rate"
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Buildings.Fluid.DXSystems.Heating.BaseClasses.Types.DefrostTimeMethods
+    defTri=Buildings.Fluid.DXSystems.Heating.BaseClasses.Types.DefrostTimeMethods.timed
+    "Type of method to trigger the defrost cycle";
+
+  parameter Real tDefRun(
+    final unit="1") = 0.5
+    "Time period fraction for which defrost cycle is run"
+    annotation(Dialog(enable=defTri == Buildings.Fluid.DXSystems.Heating.BaseClasses.Types.DefrostTimeMethods.timed));
```

Comment:
time period fraction is not clear. Explain what it is a fraction off, what is the basis, e.g., what would a value of 1 mean?